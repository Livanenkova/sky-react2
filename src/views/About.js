import {Link} from 'react-router-dom'

function About() {

  return (
    <div className="about">
      <h1>About page here</h1>
      <h3>Shop adress</h3>
      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Tempore deserunt facilis delectus suscipit sit debitis maxime repellendus quos eius nam amet, quidem ullam accusantium aspernatur porro similique quam modi voluptatum!
      </p>
      <Link to="/">Main Page</Link>
    </div>
  )
}

export default  About