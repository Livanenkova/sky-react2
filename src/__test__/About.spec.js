/* eslint-disable no-unused-vars */
/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer';
import {Router } from 'react-router-dom';
import About from '../views/About';

const { createMemoryHistory } = require("history");


it('отображается корректно', () => {
  const history = createMemoryHistory();
  const tree = renderer.create(<Router location={history.location} navigator={history}><About/></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
