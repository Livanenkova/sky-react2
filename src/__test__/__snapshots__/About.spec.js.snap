// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`отображается корректно 1`] = `
<div
  className="about"
>
  <h1>
    About page here
  </h1>
  <h3>
    Shop adress
  </h3>
  <p>
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Tempore deserunt facilis delectus suscipit sit debitis maxime repellendus quos eius nam amet, quidem ullam accusantium aspernatur porro similique quam modi voluptatum!
  </p>
  <a
    href="/"
    onClick={[Function]}
  >
    Main Page
  </a>
</div>
`;
